src/pages/project/helpers/projectRoleHelpers.ts

```diff
@@ -60,29 +70,181 @@ export const checkIfSameAffiliationOnSameRoleType = (
   return role.affiliation?.type === 'Organization' && role.affiliation?.id === newAffiliationId && isSameRoleType;
 };
 
-const findProjectManagerRoleThatHasAffiliation = (contributor: ProjectContributor, affiliationId: string) => {
-  return contributor.roles.findIndex((role) => role.affiliation?.id === affiliationId && isProjectManagerRole(role));
+const findProjectManagerRoleThatHasAffiliation = (roles: ProjectContributorRole[], affiliationId: string) => {
+  return roles.findIndex((role) => role.affiliation?.id === affiliationId && isProjectManagerRole(role));
 };
 
-const findNonProjectManagerRoleThatHasAffiliation = (contributor: ProjectContributor, affiliationId: string) => {
-  return contributor.roles.findIndex((role) => role.affiliation?.id === affiliationId && isNonProjectManagerRole(role));
+const findNonProjectManagerRoleThatHasAffiliation = (roles: ProjectContributorRole[], affiliationId: string) => {
+  return roles.findIndex((role) => role.affiliation?.id === affiliationId && isNonProjectManagerRole(role));
 };
 
 export const findRoleIndexForAffiliation = (
   asProjectManager: boolean,
-  contributor: ProjectContributor,
+  roles: ProjectContributorRole[],
   affiliationId: string
 ) => {
   return asProjectManager
-    ? findProjectManagerRoleThatHasAffiliation(contributor, affiliationId)
-    : findNonProjectManagerRoleThatHasAffiliation(contributor, affiliationId);
+    ? findProjectManagerRoleThatHasAffiliation(roles, affiliationId)
+    : findNonProjectManagerRoleThatHasAffiliation(roles, affiliationId);
 };
 
 export const notLastOfItsRoleType = (
-  contributor: ProjectContributor,
+  roles: ProjectContributorRole[],
   affiliationId: string,
   roleType: ProjectContributorType
 ) => {
-  const rolesWithoutAffiliationId = contributor.roles.filter((role) => role.affiliation?.id !== affiliationId);
+  const rolesWithoutAffiliationId = roles.filter((role) => role.affiliation?.id !== affiliationId);
   return rolesWithoutAffiliationId.some((role) => role.type === roleType);
 };
+
+export enum AffiliationErrors {
+  NO_AFFILIATION_ID,
```

Comment:
Vi har `PascalCase` for enums andre plasser nå, så kanskje vurdere det her også?
```suggestion
  NoAffiliatioId,
```

Reply:
Er det ikke en konvensjon å bruke store bokstaver på enums?

Reply:
I Java og C# er det vanlig konvensjon med store bokstaver tror jeg, men i typescript er PascalCase "standard": https://www.typescriptlang.org/docs/handbook/enums.html